### ISSUE TITLE
Updater closes but doesn't auto-restart program

### ISSUE THREAD

**Issue by TuxSH** [Ryujinx/Ryujinx#4355]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of Issue

After being prompted to update, and going through the update process I am then offered to restart the program.

Clicking on "yes" closes but does not restart the program for me.

It is worth noting I was using the latest (at time of writing) LDN build at first, but then copied the regular build over it, replacing files, then copied LDNConfig to Config (but only once).

### Reproduction Steps

Wait for a new build to be released, then automatically update and click "yes" when offered to restart.

### Log File

None relevant, files look empty.

### OS

Windows 11

### Ryujinx version

1.1.598

### Environment?

- ##### CPU: AMD Ryzen 5900x
- ##### GPU: Nvidia RTX 3060
- ##### RAM: 32 GB
- Applied Mods: N/A

**Comment by MutantAura** [comment]
Duplicate of https://github.com/Ryujinx/Ryujinx/issues/2996
